File: CSC classes/CSC106/median3.c
#include <stdio.h>

int main()
{
    int num1, num2, num3, i, j, a;
	
	printf("Enter a number: \n");
	scanf ("%d",&num1);
	
	printf("Enter a number: \n");
	scanf ("%d",&num2);
	
	printf("Enter a number: \n");
	scanf ("%d",&num3);
	
	int arr[3];
	arr[0]=num1;
	arr[1]=num2;
	arr[2]=num3;
	
	for (i = 0; i < 3; ++i)
    {
        for (j = i + 1; j < 3; ++j)
        {
            if (arr[i] > arr[j])
            {
                a =  arr[i];
                arr[i] = arr[j];
                arr[j] = a;
            }
        }
	}
	
	printf("The median of the 3 numbers is %i", arr[1]);
    return 0;
}

File: CSC classes/CSC106/Median3.java
import java.util.*;

public class Median3{
	public static void main(String[] args){
		int num1, num2, num3;
		Scanner userInput= new Scanner(System.in);
		System.out.println("Enter a number: ");
		num1= userInput.nextInt();
		
		System.out.println("Enter a number: ");
		num2= userInput.nextInt();
		
		System.out.println("Enter a number: ");
		num3= userInput.nextInt();
		
		int[] arr= {num1, num2, num3};
		
		Arrays.sort(arr);
	
		System.out.println("The median of the 3 numbers is "+arr[1]);
	}
}